--- test_build/baritone-v2/buildSrc/src/main/java/baritone/gradle/task/ProguardTask.java
+++ test_build/baritone-v2/buildSrc/src/main/java/baritone/gradle/task/ProguardTask.java
@@ -129,9 +129,6 @@
         template.add(0, "-injars '" + this.artifactPath.toString() + "'");
         template.add(1, "-outjars '" + this.getTemporaryFile(PROGUARD_EXPORT_PATH) + "'");

-        template.add(2, "-libraryjars  \"<java.home>/jmods/java.base.jmod\"(!**.jar;!module-info.class)");
-        template.add(3, "-libraryjars  \"<java.home>/jmods/java.desktop.jmod\"(!**.jar;!module-info.class)");
-        template.add(4, "-libraryjars  \"<java.home>/jmods/jdk.unsupported.jmod\"(!**.jar;!module-info.class)");

         {
             final Stream<File> libraries;
